pkg/pillar/types/downloadertypes.go

```diff
@@ -167,3 +168,71 @@ func AllowNonFreePort(gc ConfigItemValueMap, objType string) bool {
 		return false
 	}
 }
+
+// AppImgResolveConfig key/index to this is the ImageID which is allocated by the controller.
+// It will resolve container image tag to sha256
+type AppImgResolveConfig struct {
```

Comment:
I think these new types should not include "AppImg" in their names. We might only be using it for app images right now, but the mechanism has nothing to do with AppImages.

Reply:
Resolved.